index.html

```diff
@@ -226,5 +230,58 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+
+    <section id="contact" class="container">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/myyoqjqz" method="post">
+        <input
+          type="text"
+          name="name"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea
+          name="message"
+          cols="30"
+          rows="5"
+          placeholder="Enter text here"
+          maxlength="500"
+          required
+        ></textarea>
+        <button type="submit">Get in touch</button>
```

Comment:
- [ ] hi team, excellent job integrating formspree and implementing the contact form :clap: , you have done well, Just one small thing we can fix as mentioned in the project requirements **Implement the button interactions (enable, hover, etc.).**  please implement the button interactions, this will improve the UI experience you can see the button interactions from the figma :point_down: 
![image](https://user-images.githubusercontent.com/97521557/214383017-7a0cf464-37ba-4436-8c15-3ea54f7b78f4.png)